plugins/actix-web/Cargo.toml

```diff
@@ -18,6 +18,7 @@ paperclip-core = { path = "../../core", version = "0.1.0", features = ["actix"]
 paperclip-macros = { path = "../../macros", version = "0.2.0", features = ["actix"] }
 parking_lot = "0.10"
 serde_json = "1.0"
+askama = "0.9"
```

Comment:
Let's make this an optional dependency and add a feature `"swagger-ui"` below, which then depends on `askama`